Tests/SmokeXCUITests.xctestplan

```diff
@@ -44,6 +44,7 @@
         "BaseTestCase",
         "BookmarkingTests\/testAccessBookmarksFromContextMenu()",
         "BookmarkingTests\/testBookmarkingUI()",
+        "BookmarkingTests\/testBookmarksAwesomeBar()",
```

Comment:
I'm just disabling these test, but the other change their other,  there is not other change than this.
This test can be enabled again once the issue is fixed:
https://github.com/mozilla-mobile/firefox-ios/issues/13206